tests/benchmarks/docker_benchmarks/test_docker_base.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for DockerBenchmark modules."""
+
+import os
+import re
+
+from superbench.benchmarks import BenchmarkType, ReturnCode
+from superbench.benchmarks.docker_benchmarks import DockerBenchmark
+
+
+class FakeDockerBenchmark(DockerBenchmark):
+    """Fake benchmark inherit from DockerBenchmark."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name: benchmark name.
+            parameters: benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        command = os.path.join(self._args.bin_dir, self._bin_name)
+        command += " -n 'cost1: 10.2, cost2: 20.2'"
+        self._commands.append(command)
```

Comment:
same, looks strange for docker

Reply:
Forget to remove the _process() function. Thanks